vocabularies/Org.OData.JSON.V1.md

```diff
@@ -1,17 +1,190 @@
 # JSON Vocabulary
 **Namespace: [Org.OData.JSON.V1](Org.OData.JSON.V1.xml)**
 
-Terms for JSON properties
+Terms, types, and functions for JSON data
+
+OData [stream properties](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_ManagingStreamProperties) allow embedding data of arbitrary media types,
+and the OData JSON format allows [direct embedding of JSON data](https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_StreamProperty) in request and response payloads.
+
+This vocabulary defines a convenience [type for JSON data](#JSON) as well as a term for referencing a [JSON Schema](#Schema) describing the structure of the JSON data.
+
+In addition it defines two functions for [querying](#query) JSON data and using a [primitive value](#value) extracted from JSON data in common expressions, for example in `$filter`, `$orderby`, or `$compute`.
+
+**Example**
+
+The `Employees` entity set has a JSON data property `resume`:
+```json
```

Comment:
Example in the .xml schema should be (or reference) xml.

Reply:
TC decided to reference the example rather than copy it into the Vocabulary file.